app/email.go

```diff
@@ -70,7 +70,7 @@ func (a *App) SendChangeUsernameEmail(oldUsername, newUsername, email, locale, s
 		map[string]interface{}{"TeamDisplayName": a.Config().TeamSettings.SiteName, "NewUsername": newUsername})
 	bodyPage.Props["Warning"] = T("api.templates.email_warning")
 
-	if err := a.SendMail(email, subject, bodyPage.Render()); err != nil {
+	if err := a.SendNotificationMail(email, subject, bodyPage.Render()); err != nil {
```

Comment:
I see here a lot of changes from `SendMail` to  `SendNotificationMail` in functions that looks like aren't sending notifications, like the Email verification, and things like that. I'm not 100% sure, but we need a list of emails that we are sending, and know which should be sent and which not.

Reply:
My goal here was to preserve the logic and make it work as it was before (basically, disabling the notifications will disable all the emails that are pointing to `SendNotificationMail` now).

I agree on this being a good oportunity to confirm the list of things that we would like to see disabled as part of that check :)

Reply:
Yes, we can do it later anyway.